test/views/ObligationsViewSpec.scala

```diff
@@ -152,6 +158,26 @@ class ObligationsViewSpec extends TestSupport with ImplicitDateFormatter {
 
     //Income source EOPS subsection
 
+    "Show EOPS SE income source" when {
+      lazy val setup = new Setup(eopsSEIncomeSource)
+      import setup._
```

Comment:
dont need to do this, can use the setup class to set-up data as per the tests below